keras_nlp/tokenizers/tokenizer.py

```diff
@@ -129,3 +130,40 @@ def call(self, *args, mode="tokenize", training=None, **kwargs):
             raise ValueError(
                 f"Unsupported tokenizer mode. Received: mode={mode}"
             )
+
+    def recursive_utf8_decoder(self, inputs, *args, **kwargs):
+        """Recursively decodes to list of strings with 'utf-8' encoding."""
+        if str(type(inputs)) == "<class 'bytes'>":
+            inputs = inputs.decode("utf-8")
+            return inputs
+        if str(type(inputs[0])) == "<class 'bytes'>":
+            for i in range(len(inputs)):
+                inputs[i] = inputs[i].decode("utf-8")
+            return inputs
+        else:
+            for i in range(len(inputs)):
+                inputs[i] = self.recursive_utf8_decoder(
+                    inputs[i], *args, **kwargs
+                )
+
+    def detokenize_to_strings(self, inputs, *args, **kwargs):
+        """Transform detokenized inputs to strings.
```

Comment:
Detokenize, then convert the output tensor to nested lists of python strings.

This is a convenience method layered on top of `detokenize()`. This method
will call `detokenize()` and transform the output string tensors back to python
strings, by first converting output tensors to nested lists of elements, and then
converting each byte string to a python string.

Reply:
@mattdangerw just to confirm what you've written down are changes for the docstring right?

Reply:
yes!